packages/indexer/src/utils/contractUtils.ts

```
@@ -333,7 +333,11 @@ function getBetaContractAddress(
 ): string | undefined {
   // getDeployedAddressBetaRelease throws an error if the address does not exist
   try {
-    return getDeployedAddressBetaRelease(name, chainId);
+    const address = getDeployedAddressBetaRelease(name, chainId);
```

I am not sure we want to throw an error here. Originally, we expected `undefined` if a contract address does not exist.